docs/backend/cookbook/new_attribute.md

```diff
@@ -411,25 +439,30 @@ class YourAttributeDataSetQueryBuilder extends AbstractAttributeDataSetBuilder
 }
 ```
 
-Create column class `YourAttributeColumn`
-
-This file should live in `Your-module\Grid\Column` namespace and extends `\Ergonode\Grid\Column\AbstractColumn`
+Create renderer class `YourAttributeColumnRenderer`
 
 ```php
-namespace Ergonode\Grid\Column;
+namespace Ergonode\Product\Infrastructure\Grid\Column\Renderer;
```

Comment:
```suggestion
namespace YourNameSpace\Infrastructure\Grid\Column\Renderer;
```

Reply:
It's not in line with every other class entry. I can change it but than we should update every other class definition. What do you think?

Reply:
I don't know. It can cause confusion. People instead of doing it in their own module, they add commit o main repository.